Formula/simutrans.rb

```diff
@@ -32,9 +32,9 @@ class Simutrans < Formula
   uses_from_macos "curl"
   uses_from_macos "unzip"
 
-  resource "pak64" do
-    url "https://downloads.sourceforge.net/project/simutrans/pak64/122-0/simupak64-122-0.zip"
-    sha256 "ce2ebf0e4e0c8df5defa10be114683f65559d5a994d1ff6c96bdece7ed984b74"
+  resource "homebrew-pak64" do
```

Comment:
looks like `pak64` has to be in sync with `simutrans`'s version, if that is the case, we probably should remove `homebrew-` prefix (to my understanding, that is usually used for test resource, which is pretty much the same between the releases).

Reply:
Is that documented?

Reply:
That is a good question, I dont think so.